Tools/CISupport/ews-build/steps.py

```diff
@@ -1371,8 +1371,38 @@ def start(self):
         return None
 
 
-class FindModifiedLayoutTests(AnalyzeChange):
+class FindLayoutTestsBaseline(shell.ShellCommand):
+    name = 'find-layout-tests-baseline'
+    description = 'find-layout-tests-baseline running'
+    descriptionDone = 'Found baseline expectations for layout tests'
+    command = ['python3', 'Tools/Scripts/run-webkit-tests', '--print-expectations', WithProperties('--%(configuration)s')]
+
+    @defer.inlineCallbacks
+    def run(self):
+        self.log_observer = logobserver.BufferLogObserver(wantStderr=True)
+        self.addLogObserver('stdio', self.log_observer)
+
+        platform = self.getProperty('platform')
+        self.setCommand(self.command + customBuildFlag(platform, self.getProperty('fullPlatform')))
+
+        patch_author = self.getProperty('patch_author')
+        if patch_author in ['webkit-wpt-import-bot@igalia.com']:
+            self.setCommand(self.command + ['imported/w3c/web-platform-tests'])
+
+        self.setCommand(['/bin/sh', '-c', ' '.join(self.command) + ' > base-expectations.txt'])
```

Comment:
What is the reason to execute via shell, not directly?

Reply:
I think it's necessary to execute via shell to pipe stdout into a text file, correct me if I'm wrong.

Reply:
This is indeed how we achieve something like "save file on worker containing the output of a script". Although @briannafan, I would use `ShellMixin`'s `shell_command`, that way this is Windows compatible.